calico_containers/tests/st/bgp/test_route_reflector_cluster.py

```diff
@@ -32,28 +32,45 @@ def test_route_reflector_cluster(self):
              DockerHost('host3') as host3, \
              RouteReflectorCluster(2, 2) as rrc:
 
-            # Create the network on host1, but it should be usable from all
-            # hosts.
-            net = host1.create_network(str(uuid.uuid4()))
-
             # Set the default AS number - as this is used by the RR mesh, and
             # turn off the node-to-node mesh (do this from any host).
             host1.calicoctl("bgp default-node-as 64513")
             host1.calicoctl("bgp node-mesh off")
 
-            workload_host1 = host1.create_workload("workload1", network=net)
-            workload_host2 = host2.create_workload("workload2", network=net)
-            workload_host3 = host3.create_workload("workload3", network=net)
+            # Create a profile to associate with all workloads
+            host1.calicoctl("profile add TEST_GROUP")
+
+            workload_host1 = host1.create_workload("workload1")
+            workload_host2 = host2.create_workload("workload2")
+            workload_host3 = host3.create_workload("workload3")
+
+            # Add containers to Calico
+            host1.calicoctl("container add %s %s" % (workload_host1,
+                                                     DEFAULT_IPV4_ADDR_1))
+            host2.calicoctl("container add %s %s" % (workload_host2,
+                                                     DEFAULT_IPV4_ADDR_2))
+            host3.calicoctl("container add %s %s" % (workload_host3,
+                                                     DEFAULT_IPV4_ADDR_3))
+
+            # Get the endpoint IDs for the containers
+            ep1 = host1.calicoctl("container %s endpoint-id show" % workload_host1)
+            ep2 = host2.calicoctl("container %s endpoint-id show" % workload_host2)
+            ep3 = host3.calicoctl("container %s endpoint-id show" % workload_host3)
+
+            # Now add the profiles
+            host1.calicoctl("endpoint %s profile set TEST_GROUP" % ep1)
+            host2.calicoctl("endpoint %s profile append TEST_GROUP" % ep2)
```

Comment:
append will work here, but it should probably just be `set`

The profile is set or appended to the endpoint, so none of the endpoints will initially have a profile.